english/play-with-your-phone/pull-out-phone/pull-out-phone.md

```diff
@@ -0,0 +1 @@
+You open the web browser, and run into a strange site that describes perfectly about your adventure. The website's name is GitHub.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.